docs/ilm.asciidoc

```diff
@@ -0,0 +1,126 @@
+[[ilm]]
+[role="xpack"]
+== Index lifecycle management (ILM)
+
+Use the {ref}/getting-started-index-lifecycle-management.html[index lifecycle management (ILM)]
+feature in {es} to manage your APM Server indices as they age.
+ILM enables you to automate how you want to manage your indices over time,
+by automating rollovers to a new index when the existing index reaches a specified size or age.
+
+You can view and edit the index lifecycle policies in the *Index lifecycle policies* UI in {kib}.
+For more information about working with the UI,
+see {kibana-ref}/index-lifecycle-policies.html[Index lifecyle policies].
+
+APM Server currently offers two ways to get started with ILM:
+
+1. <<ilm-default,**ILM default policy**>> - Get up and running with a default index lifecycle management policy as quickly as possible.
+2. <<manual-ilm-setup,**Manual ILM**>> - Customize and manage your own ILM policies.
+
+NOTE: If you're migrating from an existing setup,
+any indices present before ILM was configured will need to be managed manually.
+
+[[ilm-default]]
+=== ILM default policy
+
+Index lifecycle management will manage an index based on its defined policy.
+The default ILM configuration applies hot and warm phase policies.
+Cold and delete phases are not defined.
+Because errors and spans lose information value faster than metrics and transactions do,
+there are two different policies defined:
+one for `errors` and `spans`, and one for `metrics` and `transactions`.
+
+[options="header"]
+|=======================================================================
+|Event type |Hot |Warm
+|Errors & Spans
+|Rollover: `max_size: 50gb`, `max_age: 1 day`,
+`priority: 100`
+|`min_age: 7 days`,
+readonly,
+`priority: 50`
+|Transactions & Metrics
+|Rollover: `max_size: 50gb`, `max_age: 7 days`,
+`priority: 100`
+|`min_age: 31 days`,
+readonly,
+`priority: 50`
+|=======================================================================
+
+IMPORTANT: Changes to the default index lifecycle policy do not take effect until the current index has rolled over.
+If you'd like to manage a custom policy, see <<manual-ilm-setup,Manual ILM>>.
+
+[float]
+[[ilm-setup]]
+==== ILM default policy setup
+
+To set up index lifecycle management, set `ilm.enabled` to `true` in apm-server.yml.
```

Comment:
I'd also mention that in case users switch between ilm enabled/disabled, they should set `setup.template.overwrite` to `true` to ensure complete setup.

Reply:
👍